apps/Minecraft Pi (Modded)/install

```diff
@@ -19,92 +14,42 @@ function error {
   exit 1
 }
 
-function install_deb {
-  echo "Installing $1..."
-  sudo dpkg -i "$1" || : # Expected To Fail
-  sudo apt-get --fix-broken install --no-install-recommends -y || error 'Unable To Complete Install'
-}
-
 ##
 ## Install mcpi-reborn
 ##
 
 if [[ "$(lsb_release -cs)" != "buster" && "$(lsb_release -cs)" != "bullseye" && "$(lsb_release -cs)" != "sid" ]]; then
-    error "Unsupported Debian Version! Your Debian version is $(lsb_release -cs), but supported versions are Buster, Bullseye, or Sid."
+    error "Unsupported RPi OS Version! Your RPi OS version is $(lsb_release -cs), but supported versions are Buster, Bullseye, or Sid. Consider searching how to upgrade your version to RPi OS Buster!"
 fi
 
 # Remove Old Minecraft Pi
 sudo apt-get remove -y minecraft-pi &>/dev/null || true
 sudo apt-get remove -y minecraft-pi-native &>/dev/null || true
 sudo apt-get remove -y mcpil-r &>/dev/null || true
```

Comment:
This is not really needed anymore, unless you are using a really old MCPIL-R and haven't upgraded since you installed it.